engine/class_modules/priest/sc_priest.cpp

```diff
@@ -1179,6 +1179,11 @@ priest_td_t::priest_td_t( player_t* target, priest_t& p ) : actor_target_data_t(
   buffs.wrathful_faerie_fermata = make_buff( *this, "wrathful_faerie_fermata", p.find_spell( 345452 ) )
                                       ->set_cooldown( timespan_t::zero() )
                                       ->set_duration( priest().conduits.fae_fermata.time_value() );
+  buffs.hungering_void_tracking = make_buff( *this, "hungering_void_tracking", p.talents.hungering_void )
+                                      ->set_quiet( true )
+                                      ->set_duration( timespan_t::from_seconds( 99 ) )
+                                      ->set_max_stack( 99 );
```

Comment:
Why 99 stacks?

Reply:
i did that just to be able to check the Nth void bolt at some point, but we dont really need it now so ill remove it